pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -217,51 +217,35 @@ func fixLibvirtServiceRunning() (bool, error) {
 
 func checkMachineDriverLibvirtInstalled() (bool, error) {
 	logging.Debugf("Checking if %s is installed", libvirtDriverCommand)
+
 	// Check if crc-driver-libvirt is available
-	path, err := exec.LookPath(libvirtDriverCommand)
+	libvirtDriverPath := filepath.Join(constants.CrcBinDir, libvirtDriverCommand)
+	err := unix.Access(libvirtDriverPath, unix.X_OK)
 	if err != nil {
+		logging.Debugf("%s not executable", libvirtDriverPath)
 		return false, err
 	}
-	fi, err := os.Stat(path)
-	if err != nil {
-		return false, err
-	}
-	// Check if permissions are correct
-	if fi.Mode()&0011 == 0 {
-		return false, errors.New("crc-driver-libvirt does not have correct permissions")
-	}
```

Comment:
I can split this change in a separate commit if needed, this changes the way we check if the binary exists/is executable, but it did not have to be part of this commit (this is the same code as in the hyperkit driver).